From 274e55a289c1f67e19c24a7672453384527247eb Mon Sep 17 00:00:00 2001
From: Tsukasa Sugiura <t.sugiura0204@gmail.com>
Date: Tue, 18 Apr 2017 21:54:12 +0900
Subject: [PATCH] Update find Boost version

Update find Boost version.
Add Boost 1.64.0 that supported Visual Studio 2017.
---
 cmake/pcl_find_boost.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/pcl_find_boost.cmake b/cmake/pcl_find_boost.cmake
index 7ccf80c727..68920ccd41 100644
--- a/cmake/pcl_find_boost.cmake
+++ b/cmake/pcl_find_boost.cmake
@@ -19,7 +19,7 @@ if(${CMAKE_VERSION} VERSION_LESS 2.8.5)
     "1.46.0" "1.46" "1.45.0" "1.45" "1.44.0" "1.44" "1.43.0" "1.43")
 else(${CMAKE_VERSION} VERSION_LESS 2.8.5)
   set(Boost_ADDITIONAL_VERSIONS
-    "1.63.0" "1.63" "1.62.0" "1.62" "1.61.0" "1.61" "1.60.0" "1.60"
+    "1.64.0" "1.64" "1.63.0" "1.63" "1.62.0" "1.62" "1.61.0" "1.61" "1.60.0" "1.60"
     "1.59.0" "1.59" "1.58.0" "1.58" "1.57.0" "1.57" "1.56.0" "1.56" "1.55.0" "1.55"
     "1.54.0" "1.54" "1.53.0" "1.53" "1.52.0" "1.52" "1.51.0" "1.51"
     "1.50.0" "1.50" "1.49.0" "1.49" "1.48.0" "1.48" "1.47.0" "1.47")
